apps/dashboard/test/system/files_test.rb

```diff
@@ -50,7 +50,7 @@ def setup
   test "copying files" do
     visit files_url(Rails.root.to_s)
     %w(app config manifest.yml).each do |f|
-      find('a', exact_text: f).ancestor('tr').click(:meta)
+      find('a', exact_text: f).ancestor('tr').find('input[type="checkbox"]').click
```

Comment:
For whatever reason - this change changed how clickable these rows are. This test was failing because this `click(:meta)` resulted in clicking the dropdown menu in the actions column - instead of clicking the row to select it.

Here's the failure image from this test. You can see the dropdown menu opened on the `manifest.yml` row.

![failures_test_copying_files](https://github.com/OSC/ondemand/assets/4874123/b4c795d6-ccfa-4300-939d-e65228826ba6)